src/Magnum/GL/AbstractShaderProgram.h

```diff
@@ -1258,19 +1258,20 @@ class MAGNUM_GL_EXPORT AbstractShaderProgram: public AbstractObject {
         AbstractShaderProgram& dispatchCompute(const Vector3ui& workgroupCount);
         #endif
 
-    protected:
         /**
-         * @brief Link the shader
+         * @brief Non-blocking linking status check
+         *
+         */
+        bool isLinkFinished();
```

Comment:
Docs still missing here ;) Cross-link with the actual GL APIs being called:

```cpp
         * @see @fn_gl_keyword{GetProgram} with 
         *      @def_gl_extension{COMPLETION_STATUS,KHR,parallel_shader_compile}
```

And mention what the fallback does if the extension isn't supported, and it'll probably be also useful to note the issue with `COMPLETION_STATUS` being false in certain scases. 

Same for `isCompileFinished()`.